cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -145,8 +145,6 @@ func (a *IngressPGReconciler) maybeProvision(ctx context.Context, hostname strin
 		a.recorder.Event(ing, corev1.EventTypeWarning, "HTTPSNotEnabled", "HTTPS is not enabled on the tailnet; ingress may not work")
 	}
 
-	logger = logger.With("proxy-group", pg)
```

Comment:
Why this? I think it's useful to see what ProxyGroup we are working against

Reply:
Oh that's my bad, I think this used to be just the name of the ProxyGroup when I added the log line